admin/tests/acceptance/user-details-personal-information-test.js

```diff
@@ -0,0 +1,48 @@
+import { module, test } from 'qunit';
+import { click, currentURL, fillIn, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { createAuthenticateSession } from 'pix-admin/tests/helpers/test-init';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Acceptance | user details personal information', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  let user = null;
+
+  hooks.beforeEach(async function() {
+    await createAuthenticateSession({ userId: 2 });
+    user = this.server.create('user');
+  });
+
+  test('visiting /users/:id', async function(assert) {
+
+    // when
+    await visit(`/users/${user.id}`);
+
+    // then
+    assert.equal(currentURL(), `/users/${user.id}`);
+  });
+
+  module('edit users details', function() {
+
+    test('should update users\'s fistName, lastName and email', async function(assert) {
+      // given
+      const user = this.server.create('user');
```

Comment:
Il y a ici deux choses qui ressemblent à des prénoms. Pour éviter cette confusion, je propose [John Doe](https://en.wikipedia.org/wiki/John_Doe). 

Il n'est pas clair quelles sont les propriétés de l'utilisateur créé par this.server.create('user'), c'est pour ça qu'il y a `updated` dans le nom, pour être sûr que la valeur soit modifiée. Mais c'est bizarre, par exemple on a l'impression que l'email n'est pas modifié. Pas moyen de spécifier les valeurs d'origine de server.create ?

Reply:
Le user factory de mirage génère des valeurs aléatoires, d'ou le serveur.create vide. Avec la nouvelle règle, j'ai override ce système pour pouvoir faire des assert cohérente en cas de patch des valeurs.

```
export default Factory.extend({
  firstName() {
    return faker.name.firstName();
  },
  lastName() {
    return faker.name.lastName();
  },
  email() {
    return faker.internet.exampleEmail().toLowerCase();
  }
});
```